src/idpi/operators/crop.py

```diff
@@ -0,0 +1,63 @@
+"""Horizontal cropping operator."""
+
+# Third-party
+import xarray as xr
+
+# Local
+from .. import metadata
+from . import gis
+
+
+def crop(field: xr.DataArray, bounds: tuple[int, ...]) -> xr.DataArray:
+    """Crop the field to the given bounds.
+
+    Only fields defined on regular grids in rotlatlon coordinates,
```

Comment:
You could raise an error message if the gridDefinitionTemplateNumber differs from 0 or 1.

Reply:
The message is contained within the field (as a byte string) but this key is not part of the geography namespace. It's possible to use earthkit data to decode the grib message however it seems a bit overkill.

Reply:
It was just an idea. One has to see that in the context of the entire logical flow, and here I do not yet have an overview.